src/anemoi/graphs/create.py

```diff
@@ -87,6 +87,32 @@ def clean(self, graph: HeteroData) -> HeteroData:
 
         return graph
 
+    def post_process(self, graph: HeteroData) -> HeteroData:
+        """Allow post-processing of the resulting graph.
+
+        This method applies any configured post-processors to the graph,
+        which can modify or enhance the graph structure or attributes.
+
+        Parameters
+        ----------
+        graph : HeteroData
+            The graph to be post-processed.
+
+        Returns
+        -------
+        HeteroData
+            The post-processed graph.
+
+        Notes
+        -----
+        Post-processors are applied in the order they are specified in the configuration.
+        Each post-processor should implement an `update_graph` method that takes and returns a HeteroData object.
+        """
+        for processor in self.config.get("post_processors", {}):
+            graph = instantiate(processor).update_graph(graph)
```

Comment:
The default value of `{}` will cause an error here. 
Should it not be a `None` and `if not None`?

Reply:
True, thanks. Default value should be `[]`.